src/EventTickets/resources/components/EventTicketsHeader.tsx

```diff
@@ -1,9 +1,9 @@
-import moment from 'moment';
+import {format} from 'date-fns';
 
 export default function EventTicketsHeader({title, startDateTime}) {
-    const fullDate = moment(startDateTime).format('dddd, MMMM Do, h:mma z');
-    const day = moment(startDateTime).format('DD');
-    const month = moment(startDateTime).format('MMM');
+    const fullDate = format(startDateTime, 'EEEE, MMMM do, hh:mmaaa');
+    const day = format(startDateTime, 'dd');
+    const month = format(startDateTime, 'EEE');
```

Comment:
Should we also be passing a local when formatting a date as a label?

See https://date-fns.org/v3.3.1/docs/I18n

Alternatively, we could localize an array of translated Month labels and do a lookup.

Reply:
I have a quick question: How should we pass the locale to the frontend? There are a few options I've been considering, like using the Fields API, adding a localization string when enqueuing the field template script, or through the Donation Form settings, all with the `get_locale()` function from WordPress. I realize this approach might not be perfect for multi-language sites since each plugin has its own way of handling languages. However, I think it could be a great starting point to get dates localized in Events. What do you think?

Reply:
Localization might be a topic for a larger team discussion. Admittedly, internationalization is not my strong suite. We can merge this as is and discuss best practices with the team.